### PR TITLE
executor: set statsLease to 0

### PR DESCRIPTION
**Pull request by hanfei1991**
fix race

### CODE REVIEW

File: executor/executor_test.go [hunk 1]
```diff
@@ -65,6 +65,7 @@ func (s *testSuite) SetUpSuite(c *C) {
 		c.Assert(err, IsNil)
 		s.store = store
 		tidb.SetSchemaLease(0)
+		tidb.SetSchemaLease(0)
```
Comment (zimulala) [inline executor/executor_test.go]: Duplicate.

Comment (coocood) [inline executor/executor_test.go]: Just one line change ...

**Comment by shenli** [comment]
Why change this?

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**